alpha/lib/model/BatchJobLockPeer.php

```diff
@@ -243,4 +244,12 @@ public static function retrieveByEntryId($entryId, array $jobTypes = null, array
 	
 	    return self::doSelect($c);
 	}
+
+	public static function retrieveByEntryIdAndStatus($obj_id, $status)
+	{
+		$c = new Criteria();
+		$c->add ( self::ENTRY_ID , $obj_id );
+		$c->add ( self::STATUS , $status );
+		return self::doSelectOne( $c );
```

Comment:
1. is default criteria applied here ? 
2. what is we have multiple batchJobs locks ?